capstone/src/main/webapp/calendar.js

```diff
@@ -0,0 +1,47 @@
+google.charts.load('current', {'packages': ['calendar', 'timeline']});
+google.charts.setOnLoadCallback(drawCharts);
+
+/**
+ * Callback function to generate charts.
+ */
+function drawCharts() {
+  const freqData = new google.visualization.DataTable();
```

Comment:
what is ```freqData```?

Reply:
`freqData` should be the calendar deadline information for the [frequency/heat map](https://user-images.githubusercontent.com/43079005/89064698-531bd000-d31f-11ea-9af3-6f7dbe73f359.png) calendar chart. If there's a more intuitive variable name, I can change it.